README.md

```diff
@@ -144,7 +144,7 @@ Or you can retrieve the solved value for human wealth normalized by permanent in
 PFexample.solution[0].hNrm
 ```
 
-For a detailed explanation of the above example please see the demo notebook [_A Gentle Introduction to HARK_](https://mybinder.org/v2/gh/econ-ark/DemARK/master?filepath=notebooks/Gentle-Intro-To-HARK-PerfForesightCRRA.ipynb).
```

Comment:
new link is this https://econ-ark.github.io/DemARK/notebooks/Gentle-Intro-To-HARK-PerfForesightCRRA.html which is interactive if you press the rocket button up top and Launch on Binder

Reply:
The new link has broken links. We'd need to replace any links to econ.jhu with econ2.jhu. Also would it be possible to reorder the examples there, so that there is more of a progression?

Reply:
Ah yeah. That sounds like a bigger lift than I thought. 

You can make an issue about the broken links here https://github.com/econ-ark/demark and someone can take care of it later or if you have time and know how to use jupyter / want to venture into learning jupyter, you are welcome to make a PR there as well. 

Thanks for reviewing all this!